Assets/MRTK/Examples/Experimental/ColorPicker/Scripts/ColorPicker.cs

```diff
@@ -0,0 +1,352 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using TMPro;
+using Microsoft.MixedReality.Toolkit.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.ColorPicker
+{
+    /// <summary>
+    /// Example script to demonstrate adding sliders to control material values at runtime.
```

Comment:
Is this script a duplicate of the one in the SDK folder?

Reply:
Good catch! It was a dupe, and now has been removed.